### ISSUE TITLE
Collect data for search autocomplete

### ISSUE THREAD

**Issue by emtwo** [mozilla/activity-stream#709]
This issue will be closely associated with #403. 

Tracking the index of the chosen item would be nice (universal search does this too) or -1 if no items from the suggestions drop down is chosen.

We could also track how many characters were typed before hitting "enter" or choosing an item.

Are we allowed to collect the search string? That would be cool.

**Comment by pdehaan** [comment]
Blocked by #403

**Comment by sarracini** [comment]
@emtwo search stuff landed so we can start collecting metrics, let me know what you want to collect and i can get started on this

**Comment by emtwo** [comment]
Let's collect the search index and number of characters typed for now.

**Comment by sarracini** [comment]
Perhaps it would be valuable to see if they're searching for stuff they've previously searched for? Might be useful in understanding how people search for content they've searched for before. @emtwo

**Comment by emtwo** [comment]
That would indeed be pretty great to know! I'm not familiar with the current implementation of search but my guess would be we don't keep track of their previous searches. If that's the case, it's worth some discussion (and a separate issue) about how we would keep track of user searches.

If we're already keeping track of that then I'd say go for it! Maybe we could send a counter for how many times they've searched certain keywords before.

**Comment by sarracini** [comment]
So search has something called "form history" which have access to. I can just track if the thing they searched for is part of the form history or not, without tracking the actual string itself. Thoughts?

**Comment by sarracini** [comment]
Closing this for now until we can find a better way to collect things around search
